test/smoke/testApps/OpenTelemetryApiSupport/src/main/java/com/microsoft/ajl/simple/SpringBootApp.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.ajl.simple;
```

Comment:
copyright?

Reply:
none of the smoke tests have copyright headers, if we want to change we should do across all